### ISSUE TITLE
Revert changes in submitting feedback on Android

### ISSUE THREAD

**Issue by simonrozsival** [toggl/mobileapp#2857]
## 💥 Crash Report
Tapping the `Submit feedback` button in settings crashes the app.

### 👥 Impact
This bug is not in any public release but it must be resolved before any future releases.

### 🔗 Context
This was caused by the changes in iOS app where we now have the custom feedback view (released in 1.8 if I am not mistaken).

The fastest way of fixing this would be to revert the changes to the feedback sending mechanism for Android - use the old way of sending emails. The custom feedback view is scheduled for the 1.5 release (see #2704) which will replace this soon-ish.

**Comment by amulware** [comment]
Yes, please revert the changes relevant to make this work again. We'll do the android custom feedback stuff down the line, it's too much work to wait for now, and the design isn't even final.
